api/docs/release.dox

```diff
@@ -227,7 +227,6 @@ Further non-compatibility-affecting changes include:
  - Added instr_convert_to_isa_regdeps() API that converts an #instr_t from a real ISA
    (e.g., #DR_ISA_AMD64) to the #DR_ISA_REGDEPS synthetic ISA.
 
```

Comment:
Do we need to document the fact the ger_register_name() now uses the global context to determine what name to return? (virtual or real ISA register)

Reply:
I think the local note is enough, because the default behavior won't change as the dcontext ISA should be set to the expected default.

Reply:
Ok!